src/app/CommandPathParams.h

```diff
@@ -19,6 +19,7 @@
 #pragma once
 
 #include <app/util/basic-types.h>
+#include "lib/core/GroupId.h"
```

Comment:
`basic-types.h` includes `lib/core/DataModelTypes.h`, which includes `GroupId.h` already. So this isn't necessary.

Reply:
It isn't necessary to compile, but it's necessary to satisfy IWYU checkers (which includes our internal tools). There is no harm, we should land changes like this if they help SDK consumers satisfy their own tooling needs.

Reply:
Include what you use seems a reasonable requirement. We should not rely on transitive includes.